features/all.yml

```diff
@@ -0,0 +1,6 @@
+name: all
+description: The `all` CSS property resets all CSS properties, except for `direction` and `unicode-bidi`. Only the global CSS keywords for explicit defaulting (`initial`, `inherit`, etc.) are allowed here, since they are the only values supported on all CSS properties.
```

Comment:
```suggestion
description: The `all` CSS property is a shorthand for all CSS properties, except for `direction` and `unicode-bidi`. It accepts only the keywords for explicit defaulting (such as `initial` and `inherit`), since they are the only values supported on all CSS properties.
```